core/vm/actor/builtin/v0/init/init_actor.cpp

```diff
@@ -23,8 +23,9 @@ namespace fc::vm::actor::builtin::v0::init {
 
   ACTOR_METHOD_IMPL(Construct) {
     OUTCOME_TRY(runtime.validateImmediateCallerIs(kSystemActorAddress));
-    InitActorState state{};
-    state.network_name = params.network_name;
```

Comment:
`InitActorState` implements [`Ipld::Visit`](https://github.com/filecoin-project/cpp-filecoin/blob/f42d38b6b24a010a4a7dac2e1f3bf59bdfc5319d/core/vm/actor/builtin/v0/init/init_actor.hpp#L59-L65).
Use `IpldPtr{runtime}->load(state)` to initialize amt/hamt/cidt/etc fields in constructor (other methods call `getCurrentActorStateCbor` which calls `load`)

Reply:
ok